app/models/concerns/project_media_creators.rb

```diff
@@ -49,6 +49,21 @@ def create_annotation
     end
   end
 
+  def create_mt_annotation
+    bot = Bot::Alegre.default
+    if !bot.nil? && bot.should_classify?(self.text)
+      languages = self.project.settings[:languages] unless self.project.settings.nil?
```

Comment:
I suggest to replace this line by `languages = self.project.get_languages`, which already handles if settings are not present (among other things) and makes the code simpler.